Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -555,6 +554,25 @@ private class SendCrashReportsSetting: Setting {
     }
 }
 
+private class PrivacyPolicySetting: Setting {
```

Comment:
To address this point, you can create a private helper method in the file scope that sets up and pushes a SettingsContentViewController.